gwpy/signal/qtransform.py

```diff
@@ -243,7 +244,7 @@ def __iter__(self):
         Yields a `QTile` at each frequency
         """
         # for each frequency, yield a QTile
-        for freq in self._iter_frequencies():
+        for freq in unique_everseen(self._iter_frequencies()):
```

Comment:
The same result can be achieved using something like this with no extra dependencies:

https://github.com/gwpy/gwpy/blob/v0.15.0/gwpy/cli/cliproduct.py#L94-L102

Reply:
@duncanmmacleod, good call, I've patched this to import `unique` from `gwpy.cli.cliproduct` (which I wasn't aware we had til you pointed it out). Note there is one small catch: a module-level import would be circular, so the import must be done from inside the two class methods that use this function.